src/sinopiaApi.js

```diff
@@ -40,6 +40,18 @@ export const fetchResource = (uri, isTemplate) => {
     })
 }
 
+// Fetches list of groups
+export const getGroups = () => {
+  fetch(`${Config.sinopiaApiBase}/groups`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((resp) => checkResp(resp)
+      .then(() => resp.json()))
+}
+
```

Comment:
Not sure why this isn't working.  It is very similar to this function: https://github.com/LD4P/sinopia_editor/blob/main/src/sinopiaApi.js#L113-L121